docs/more-info/build-server-setup/octopus-deploy.md

```diff
@@ -1,25 +1,36 @@
 # Octopus deploy
 While not a build server, there are a few things to consider when using Octopus Deploy with GitVersion.
 
- - GitVersion follows [continuous delivery](../../reference/continuous-delivery.md) versioning by default
-   - This means builds will keep producing *the same version* with just metadata differing.
-     If you try to push every build into Octopus you will have issues
+ GitVersion follows [continuous delivery](../../reference/continuous-delivery.md) versioning by default. This means builds will keep producing *the same version* with just metadata differing. For example, when you start a new release (say `1.0.0`) with git flow, the branch will start with a semver like `1.0.0-beta.1+0`, and the Octopus Nuget package will have a version of `1.0.0-beta0001`. As you commit changes to this release branch the *metadata* of the semver will increase like so: `1.0.0-beta.1+1`, `1.0.0-beta.1+2`, etc. However, the version of the corresponding Octopus Nuget package will retain the *same* `1.0.0-beta0001` version you started with. The problem is Octopus will prevent you from deploying these revisions because it sees the same Nuget package version and thinks nothing has changed.
 
-Because Octopus uses NuGet under the covers you cannot publish every build into Octopus deploy. For this we have two possible options:
+This limitation means you cannot continue to push revisions in this manner without some intervention (or changes to GitVersion's configuration). To work around this problem we have two possible options:
 
 ## Solutions
-### 'Release' packages to Octopus deploy
-Rather than all builds going into Octopus's NuGet feed, you release builds into it's feed. When you push a package into the NuGet feed you need to tag that release. The next commit will then increment the version.
+### Manually publish your 'release' branch to Octopus deploy
```

Comment:
This doesn't have to be a manual process.  Most CI servers, for example, TeamCity, allow you to trigger a build when a tag is applied to a repository, so the deployment to the Octopus Deploy Server can still be automatic.